engine/common/launcher.c

```diff
@@ -91,9 +91,13 @@ _inline int Sys_Start( void )
 #if !XASH_WIN32
 int main( int argc, char **argv )
 {
+#if XASH_PSVITA
+	// argv is normally empty, so we generate our own
```

Comment:
You could just copypaste the whole main and Sys_Start, and disable compiling this file whatsoever. But ok, it will be split later then.